backend/Dockerfile

```diff
@@ -1,4 +1,40 @@
-FROM l.gcr.io/google/bazel:0.24.0 as builder
+FROM ubuntu:18.04 as builder
```

Comment:
Can you briefly explain why we need to switch to ubuntu?

Reply:
Yes of course, because the original bazel builder image only supports x86_64, and to make it support multi-arch, I used ubuntu and installed bazel to make the new builder image do the same job as original builder image.

If there are any better suggestions about this, please do point it out and I will change.

Reply:
Thanks, sounds good to me.

Reply:
Awesome! Can we then test it? :)